OTKit/otkit-web-typography/token.yml

```diff
@@ -0,0 +1,74 @@
+props:
+  otkit-web-heading-xlarge-font-size:
+    type: "size"
+    value: "{!font-size-xlarge}"
+  otkit-web-heading-xlarge-font-weight:
+    type: "raw"
+    value: "{!font-weight-bold}"
+  otkit-web-heading-xlarge-line-height:
+    type: "size"
+    value: "{!line-height-xlarge}"
+
+  otkit-web-heading-large-font-size:
+    type: "size"
+    value: "{!font-size-large}"
+  otkit-web-heading-large-font-weight:
+    type: "raw"
+    value: "{!font-weight-bold}"
+  otkit-web-heading-large-font-weight-normal:
+    type: "raw"
+    value: "{!font-weight-normal}"
+  otkit-web-heading-large-line-height:
+    type: "size"
+    value: "{!line-height-large}"
+
+  otkit-web-heading-medium-font-size:
+    type: "size"
+    value: "{!font-size-medium}"
+  otkit-web-heading-medium-font-weight:
+    type: "raw"
+    value: "{!font-weight-medium}"
+  otkit-web-heading-medium-line-height:
+    type: "size"
+    value: "{!line-height-medium}"
+
+  otkit-web-subheading-small-font-size:
+    type: "size"
+    value: "{!font-size-small}"
+  otkit-web-subheading-small-font-weight:
+    type: "raw"
+    value: "{!font-weight-medium}"
+  otkit-web-subheading-small-line-height:
+    type: "size"
+    value: "{!line-height-small}"
+
+  otkit-web-bodytext-small-font-size:
+    type: "size"
+    value: "{!font-size-small}"
+  otkit-web-bodytext-small-font-weight:
+    type: "raw"
+    value: "{!font-weight-normal}"
+  otkit-web-bodytext-small-line-height:
+    type: "size"
+    value: "{!line-height-small}"
+
+  otkit-web-subtext-xsmall-font-size:
+    type: "size"
+    value: "{!font-size-xsmall}"
+  otkit-web-subtext-xsmall-font-weight:
+    type: "raw"
+    value: "{!font-weight-medium}"
+  otkit-web-subtext-xsmall-font-weight-normal:
+    type: "raw"
+    value: "{!font-weight-normal}"
+  otkit-web-subtext-xsmall-line-height:
```

Comment:
how are you handling the size differences in mobile versus full web, or are you gonna handle that later?

Reply:
@francesyun told me that the invision typography design spec I linked above is only true for desktop. Mobile is outdated, so I am not even attempting to solve that problem now. This token is only meant for full web. I can change the platform name to `desktop` to make it clear?

Reply:
Or maybe `fullweb`

Reply:
Nevermind `fullweb` is horrible, `web-desktop`? I don't have a strong preference

Reply:
Yes, let's adjust the name to have `desktop`.

Reply:
OK